src/aws-elasticache/daily-snapshot-time.ts

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
`daily-snapshot-time.ts` is almost entirely reused from fsx's [maintenance-time.ts](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-fsx/lib/maintenance-time.ts).